Login.js

```diff
@@ -0,0 +1,58 @@
+const registrationForm = document.getElementById("registrationForm");
+registrationForm.addEventListener("submit", async function (event) {
```

Comment:
why you make this function async?

Reply:
code is modified

this is not required here, actually, I just copied and pasted some of the code section from my previous task where I used promises while I was applying MockApi on the form.